inventory/views.py

```diff
@@ -105,3 +122,20 @@ def add_expiry_post(request):
             return Response({"errors": "serialise fail"}, status=400)
     else:
         return Response({"error": "Invalid request method"}, status=405)
+
+
+@api_view(["POST"])
+@permission_classes([IsAuthenticated])
+def revert_order(request):
+    try:
+        target_id = request.data.get("id")
+        if target_id is None:
+            return Response({"error": "Invalid request body"}, status=400)
+        try:
+            instance = Order.objects.get(pk=target_id)
```

Comment:
note: to return previously withdrawn or deposited items, can refer to notion for cases

just a note understand its a WIP